daily-coding-challenges/26-05-2020/findGreatestNumInArray.js

```diff
@@ -4,4 +4,11 @@
 */
 const findGreatestNumInArray = (arr) => {
 	/* Your implementation goes here */
-}
\ No newline at end of file
+	let maxNum = arr[0];
```

Comment:
@iamfortune, what happens if `arr` is `null` or `undefined`? Is there a way you could rewrite this to handle the edge case?

Reply:
Sure, going to try sir

Reply:
Great @iamfortune. I hope you understand what I'm asking to do here. Please let me know if you don't or if you have any questions

Reply:
you want me to write the algorithm in a way that it recognizes and probably returns -1 or wouldn't work if array is null (empty) or not defined? yeah?